src/bin/pg_autoctl/monitor.c

```diff
@@ -1518,65 +1529,60 @@ printCurrentState(void *ctx, PGresult *result)
 		return;
 	}
 
-	for (int i = 0; i <= maxHostNameSize; i++)
+	for (int i = 0; i < maxNameSize; i++)
 	{
-		if (i < maxHostNameSize)
-		{
-			nameSeparatorHeader[i] = '-';
-		}
-		else
-		{
-			nameSeparatorHeader[i] = '\0';
-		}
+		nameSeparatorHeader[i] = '-';
 	}
 
 	if (pg_autoctl_debug)
 	{
-		fformat(stdout, "%*s | %6s | %5s | %5s | %17s | %17s | %8s | %6s\n",
-				maxHostNameSize, "Name", "Port",
+		fformat(stdout, "%*s | %6s | %5s | %17s | %17s | %8s | %6s\n",
+				maxNameSize, "Name",
 				"Group", "Node", "Current State", "Assigned State",
 				"Priority", "Quorum");
 
-		fformat(stdout, "%*s-+-%6s-+-%5s-+-%5s-+-%17s-+-%17s-+-%8s-+-%6s\n",
-				maxHostNameSize, nameSeparatorHeader, "------",
-				"-----", "-----", "-----------------", "-----------------",
+		fformat(stdout, "%*s-+-%6s-+-%5s-+-%17s-+-%17s-+-%8s-+-%6s\n",
+				maxNameSize, nameSeparatorHeader,
+				"------", "-----", "-----------------", "-----------------",
 				"--------", "------");
 	}
 	else
 	{
-		fformat(stdout, "%*s | %6s | %5s | %5s | %17s | %17s\n",
-				maxHostNameSize, "Name", "Port",
+		fformat(stdout, "%*s | %6s | %5s | %17s | %17s\n",
+				maxNameSize, "Name",
 				"Group", "Node", "Current State", "Assigned State");
 
-		fformat(stdout, "%*s-+-%6s-+-%5s-+-%5s-+-%17s-+-%17s\n",
-				maxHostNameSize, nameSeparatorHeader, "------",
-				"-----", "-----", "-----------------", "-----------------");
+		fformat(stdout, "%*s-+-%6s-+-%5s-+-%17s-+-%17s\n",
+				maxNameSize, nameSeparatorHeader,
+				"------", "-----", "-----------------", "-----------------");
 	}
 	free(nameSeparatorHeader);
 
 	for (currentTupleIndex = 0; currentTupleIndex < nTuples; currentTupleIndex++)
 	{
-		char *hostname = PQgetvalue(result, currentTupleIndex, 0);
-		char *nodeport = PQgetvalue(result, currentTupleIndex, 1);
-		char *groupId = PQgetvalue(result, currentTupleIndex, 2);
-		char *nodeId = PQgetvalue(result, currentTupleIndex, 3);
-		char *currentState = PQgetvalue(result, currentTupleIndex, 4);
-		char *goalState = PQgetvalue(result, currentTupleIndex, 5);
-		char *candidatePriority = PQgetvalue(result, currentTupleIndex, 6);
-		char *replicationQuorum = PQgetvalue(result, currentTupleIndex, 7);
+		char *name = PQgetvalue(result, currentTupleIndex, 0);
+
+		/* char *hostname = PQgetvalue(result, currentTupleIndex, 1); */
+		/* char *nodeport = PQgetvalue(result, currentTupleIndex, 2); */
```

Comment:
I think it's still very useful to print these in the state table. It's quite common to want to know where the name points to. Especially in cases where the default names are used, since they are not very useful.

Reply:
I'm trying to stay within the 80 columns width if that's possible. The JSON output has all the information. Do you think it's important enough to have host:port on the `pg_autoctl show state` output to have very long lines?

Reply:
The hostname for sure. I think it's very common to want to connect to a node that is in the wrong state. The port I can see not being super useful for most production users, but I don't think that matters much in linesize.